tests/test_agg_mds_adapters.py

```diff
@@ -5654,3 +5654,1201 @@ def test_get_metadata_icdc():
             },
         },
     }
+
+
+@respx.mock
+def test_get_metadata_gdc():
```

Comment:
this test file is huge, can we split these tests into one file per adapter?

Reply:
I did split them into multiple files.

Reply:
thank you!

Reply:
I'll let Jian and Craig review the actual code changes 🙂